package.json

```diff
@@ -65,7 +65,7 @@
         "html2canvas": "1.0.0-rc.7",
         "mobile-detect": "1.4.3",
         "nanoid": "3.1.31",
-        "qrcode": "1.4.4",
+        "qrcode": "^1.4.4",
```

Comment:
Why so? This will install `1.5.0` version in Surf (while keeping @types/qrcode: "1.4.0")